integration/fuse/src/main/java/alluxio/fuse/AlluxioJniFuseFileSystem.java

```diff
@@ -402,7 +402,7 @@ private int readInternal(String path, ByteBuffer buf, long size, long offset, Fu
         if (offset - is.getPos() < is.remaining()) {
           is.seek(offset);
           while (rd >= 0 && nread < sz) {
```

Comment:
The same question, is this while loop necessary? @LuQQiu @apc999

Reply:
After an offline discussion with @LuQQiu , I knew the the reason is that we want keep read a specific size of input stream or touch the EOF, when the actual read length is smaller that the specific length, using while-read mode can continue to read the inputstream.